tests/unit/prometheus_alert_rules/with_template_string_and_unit.rule

```diff
@@ -0,0 +1,9 @@
+alert: PrometheusTargetMissing
```

Comment:
Re filename: I think we call the %% literal "topology stub", not "template string".

Reply:
That doesn't make it correct :)
What is "template string"? Is it the topology stub or something else?

Reply:
Yes it is the topology string. I'm not going to change Balbir's naming scheme here. If you think it is important, open a new PR